tools/system_libs.py

```diff
@@ -690,7 +691,7 @@ def get_files(self):
         'getgrouplist.c', 'initgroups.c', 'wordexp.c', 'timer_create.c',
         'faccessat.c',
         # 'process' exclusion
-        'fork.c', 'vfork.c', 'posix_spawn.c', 'execve.c', 'waitid.c', 'system.c'
+        'fork.c', 'vfork.c', 'posix_spawn.c', 'posix_spawnp.c', 'execve.c', 'waitid.c', 'system.c'
```

Comment:
is this addition a drive-by fix, or related to the PR?

Reply:
Its required.. this file contains an umimplementd syscall (same as posix_spawn.c) which results in an undefined symbols if you force include the whole of libc (which is what MAIN_MODULE=1 does).